modules/project/variables.tf

```diff
@@ -292,12 +292,24 @@ variable "shared_vpc_host_config" {
 
 variable "shared_vpc_service_config" {
   description = "Configures this project as a Shared VPC service project (mutually exclusive with shared_vpc_host_config)."
-  # the list of valid service identities is in service-accounts.tf
+  # the list of valid service identities is in service-agents.yaml
   type = object({
-    host_project         = string
-    service_identity_iam = optional(map(list(string)))
+    host_project           = string
+    service_identity_iam   = optional(map(list(string)), {})
+    grant_iam_for_services = optional(list(string), [])
```

Comment:
can we rename to `service_identity_iam_grants`?

Reply:
Sound good until I realize that I have `service_identity_iam` just above. Maybe `services_iam_grants`?

Reply:
`service_iam_grants` ? :)

Reply:
Thanks